java-symbolic-execution/src/main/java/org/sonar/java/se/checks/XxeProcessingCheck.java

```diff
@@ -188,7 +189,7 @@ private static boolean documentBuilderFactoryIsSecured(@Nullable ConstraintsByDo
       .names("setAttribute").addParametersMatcher(JAVA_LANG_STRING, JAVA_LANG_OBJECT).build(),
     MethodMatchers.create().ofSubTypes(XML_INPUT_FACTORY, SAX_PARSER, SCHEMA_FACTORY, VALIDATOR, XML_READER, SAX_BUILDER)
       .names("setProperty").addParametersMatcher(JAVA_LANG_STRING, JAVA_LANG_OBJECT).build(),
-    MethodMatchers.create().ofSubTypes(DOCUMENT_BUILDER_FACTORY, SAX_PARSER_FACTORY, XML_READER, SAX_BUILDER, SAX_READER)
+    MethodMatchers.create().ofSubTypes(DOCUMENT_BUILDER_FACTORY, SAX_PARSER_FACTORY, TRANSFORMER_FACTORY, SCHEMA_FACTORY, XML_READER, SAX_BUILDER, SAX_READER)
```

Comment:
Isn't this change also having an impact on rule S2755?

Reply:
In theory yes, we now do not report an issue if you set the correct attributes for S2755 via "setFeature".
In practice, I expect that the code will throw an exception at runtime.
In any case, other factories are also subject to this behavior, I think it is fine to keep it as it is.